ecl/hql/hqlmanifest.cpp

```diff
@@ -240,6 +266,7 @@ void ResourceManifest::addToArchive(IPropertyTree *archive)
                 const char *filepath = item.queryProp("@originalFilename");
                 resTree->setProp("@originalFilename", filepath);
                 resTree->setProp("@resourcePath", respath);
+                queryJfrog(filepath, item);                
```

Comment:
This doesn't need to write the resource to a known filename - it could read it into the content.  Otherwise there is the issue of cleaning up the downloads.

Reply:
I am not sure what you mean by this. The only way to access the contents in jfrog artifactory is to download the files.